modules/fluid_properties/fluid_properties.mk

```diff
@@ -0,0 +1,7 @@
+# Saline
+SALINE_DIR ?= $(APPLICATION_DIR)/contrib/saline
```

Comment:
this should be fluid properties module instead of APP_DIR ?

Reply:
Hm. I can double check when I'm back. I thought that get changed multiple times during build to whatever it needs to be...

Reply:
this should be changed to
```suggestion
SALINE_DIR ?= $(MOOSE_DIR)/modules/fluid_properties/contrib/saline
```